modules/api-support-tiers.adoc

```diff
@@ -10,7 +10,7 @@ All commercially supported APIs, components, and features are associated under o
 [discrete]
 [id="api-tier-1_{context}"]
 == API tier 1
-APIs and application operating environments (AOEs) are stable within a major release for a minimum of 12 months or 3 minor releases from the announcement of deprecation, whichever is longer. The release that introduces a new or revised API or AOE, and the two following minor releases (n, n+1, n+2).
+APIs and application operating environments (AOEs) are stable within a major release.  They may be deprecated within a major release, but they will not be removed until a subsequent major release.
```

Comment:
I think extra space in this line  after "release."

Reply:
others are lgtm.

Reply:
removed, not that it matters since once rendered all spacing is unified.